src/test/java/seedu/address/stubs/FindCommandParserStub.java

```diff
@@ -17,6 +18,7 @@
      */
     public static Name extractName(String args) throws ParseException {
```

Comment:
There may not be any usages of this method after the current changes, may not need `FindCommandParserStub.java` anymore if `extractName()` is not used. But ok to leave in if there may be future tests that require it.